src/versionbits.cpp

```diff
@@ -105,6 +104,42 @@ ThresholdState AbstractThresholdConditionChecker::GetStateFor(const CBlockIndex*
     return state;
 }
 
+// return the numerical statistics of blocks signalling the specified BIP9 condition in this current period
+BIP9Stats AbstractThresholdConditionChecker::GetStateStatisticsFor(const CBlockIndex* pindexPrev, const Consensus::Params& params) const
+{
+    BIP9Stats obj;
+    
+    int elapsedBlocks;
+    int nPeriod = Period(params);
+    int nThreshold = Threshold(params);
```

Comment:
`nPeriod` and  `nThreshold` are used again in line 137 (now line 135) to calculate the `possible` value. But you are right about `elapsedBlocks` and the actual `bool possible`, so I took those declarations out and simplified the assignments.